frontstage/common/session.py

```diff
@@ -73,12 +59,8 @@ def get_decoded_jwt(self):
     def get_expires_in(self):
         return jwt.decode(self.encoded_jwt_token)["expires_in"]
 
-    def is_persisted(self):
-        return self.persisted
-
-    def save(self):
+    def set(self):
```

Comment:
This is probably an acceptable change given the single (Redis) backend implementation. I'm guessing this was intended to be a generic "save" interface that could then have any backend library implementation. But they never got that far.

Reply:
It's not ideal and I believe setex is depreciated, however it wasn't my intention to hang around on this bit, make it align marginally better and get a story up to look to migrate to set with EX attribute